src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -46,6 +48,21 @@ export const RichTextEditor: FC<RichTextEditorProps> = ({
         onBlur: () => onBlur && onBlur(JSON.stringify(localValue.current)),
     };
 
+    useMemo(() => {
```

Comment:
you could use a `useCallback` instead, since the dependency is mutable: see https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node